### ISSUE TITLE
nightfin fishing is broken on this server

### ISSUE THREAD

**Issue by pwpwpwpw** [slowtorta/turtlewow-bug-tracker#6355]
I've been trying to fish nightfin and it has to be bugged, I tried WPL the north stream/river and the south big lake where scholo is at 22:00 server time and 02:00 server time, 96%+ redgil, a trout and nothing else and I had previous records of 10% nightfin with fishingbuddy at these hours/location. 

I tried the same thing in winterspring at 02:00-04:00 server time, 95%+ whitescale salmon in 2 lakes and nothing else, both are supposed to have nightfin. Also, in hinterlands, tried at 22:00 server time and 02:00, same thing, 95% redgil, no nightfin, all in all these must have been at least 400 catches without a single nightfin, they are supposed to drop 20% or so and nothing.

Item: https://database.turtle-wow.org/?item=13759

**Comment by YamaYAML** [comment]
Yep! I have found NO window at all for Nightfin Fish since the day of time changes were implemented.

**Comment by pwpwpwpw** [comment]
> I started making this, didn't get too far but maybe it helps ![Nightfin](https://private-user-images.githubusercontent.com/133578511/307628274-eecc8f28-e0dc-49b8-bccc-198d2ecec1b1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg5MTUxMDQsIm5iZiI6MTcwODkxNDgwNCwicGF0aCI6Ii8xMzM1Nzg1MTEvMzA3NjI4Mjc0LWVlY2M4ZjI4LWUwZGMtNDliOC1iY2NjLTE5OGQyZWNlYzFiMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyNlQwMjMzMjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MTJjNjUxN2Y5NWZkNmY4MzQ1NDRkMTFkNTA5ZWEwZDRlYWU4YzQ0ZWM1NzQ0YTU0NmFlNzRlYjQ0ZWRhZjgxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.3-ocDcMdrKCkK2L-wf2j-Vpjv6ir89tzipqR8lQLUMc)

can you try doing 30min rounds? 10min is too low, also where in wpl exactly you found nightfin?

**Comment by Jallarial** [comment]
West of Andorhal, in the lake between the entrance to Undead area and the Alliance camp, was where I went.

But no I won't be able to do anymore work on this. But I found with 10mins it's almost immediately obvious whether you are going to get some or none.
